.travis.yml

```diff
@@ -7,14 +7,14 @@ sudo: false
 env:
   global:
     - TESTS=false
-    - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git"
+    - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git#easy"
```

Comment:
do you still need this now that the changes have been released?

Reply:
no, this is no longer needed

Reply:
can you rebase your change to master? It'd be nice to remove the deprecation warnings in that repo :-)